charts/caraml/Chart.yaml

```diff
@@ -72,4 +72,4 @@ maintainers:
   name: caraml-dev
 name: caraml
 type: application
-version: 0.8.15
+version: 0.8.16
```

Comment:
does this have to be changed manually?
[latest](https://github.com/caraml-dev/helm-charts/blob/main/charts/caraml/Chart.yaml#L75) is 0.8.20 now